dashboards/ds-solarwinds-webapi/readme.md

```diff
@@ -0,0 +1,12 @@
+# SolarWinds Orion Nodes `API`
+
+This dashboard gives an overview of nodes alongside summary of heatlh from SolarWinds Orion using the powerful Web API and dashboard sharing capabilities of SquaredUp. 
+
+## Requirements
+
+- Copy the images repository on the SquaredUp IIS folder (or individually if the images repository already exists)
+- Create a WebAPI (Basic Auth) provider under Settings > Integrations. Name it exactly “Solarwinds” (if you name it differently, you will need to select the provider manually for each tile on the dashboard). Fill in all the required details to connect to your own SolarWinds environment (url, username, password, ignore invalid ssl, etc).
+- Create a new dashboard, select the <> on the top right and copy the content of the dashboard.json file and click Apply Changes
+ 
```

Comment:
Maybe call out updating the `rowLink` here too

Reply:
Do we want to detail the provider side of this at all? We skipped this during our discussion on Friday.